### ISSUE TITLE
Machine Learning intern model reviews-catalog

### ISSUE THREAD

**Issue by Eamymao** [Eamymao/winter-ann-arbor#2]
## Summary: 
To prepare for the interview for machine Learning intern, I collected the introduction of several relevant models and the appropriate conditions to apply the corresponding models.

### ML Models
- Random Forest
[https://github.com/Eamymao/winter-ann-arbor/issues/1](url)
- GBDT: Gradient Boosting Decision Tree
- XGBoost
- Logistics Regression
- Factorization Machines(recommend system)

### DL Models
- LSTM 
- CNN 

### Other Machine Learning topics
- optimizer
- solutions to overfitting
- regularization terms and weights shrinkage
